.github/dependabot.yml

```diff
@@ -8,7 +8,7 @@ updates:
     ignore:
       - dependency-name: "*"
         update-types:
-          ["version-update:semver-major", "version-update:semver-patch"]
+          ["version-update:semver-major", "version-update:semver-minor", "version-update:semver-patch"]
```

Comment:
I'm confused -- won't this ignore all version updates? (and same question for the below `ignore.update-types` sections)

Reply:
Oh duh!  You're right it's ignore, not include XD  I'll fix that.